fe1-web/src/features/wallet/screens/__tests__/__snapshots__/WalletSingleRollCall.test.tsx.snap

```diff
@@ -356,6 +356,8 @@ exports[`ViewSingleRollCallScreenHeader renders correctly 1`] = `
                           <View>
                             <View>
                               <qrcode
+                                overlayText="Scan to get
+user token"
                                 value="{"pop_token":"oKHk3AivbpNXk_SfFcHDaVHcCcY8IBfHE7auXJ7h4ms="}"
```

Comment:
Are you sure it is normal that pop token values disappeared from snapshots?

Reply:
They did not disappear. 

GitHub identified a potential syntax error and render it in dark red. [More info,](https://stackoverflow.com/questions/30667675/what-does-a-red-text-background-mean-in-github-comparison-view)

Reply:
Okay, sorry!